src/libm-tester/CMakeLists.txt

```diff
@@ -100,6 +114,18 @@ macro(test_extension SIMD)
 	      if (MPFR_INCLUDE_DIR)
 		target_include_directories(${T} PRIVATE ${MPFR_INCLUDE_DIR})
 	      endif()
+
+	      set(T "tester2y${SCSIMD}${P}")
```

Comment:
Same here, make sure the distinction between the old tester2 and tester2y is clear